### PR TITLE
Update license with Apache v2

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
Added Copyright based on Apache (CELA requested)
Removed unnecessary space.
Removed personal information
Added comment for future code refactoring

### CODE REVIEW

File: AutomationManager.ps1 [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+# Licensed under the Apache License.
 # Description : This script manages all the setup and test operations in Azure environemnt.
 #               It is an entry script of Azure Automation
 # Operations :
 #              - Installing AzureSDK
 #              - VHD preparation : Installing packages required by ICA, LIS drivers and waagent
 #              - Uplaoding test VHD to cloud
 #              - Invokes azure test suite
-## Author : v-shisav@microsoft.com, v-ampaw@microsoft.com, lisasupport@microsoft.com
+## Author : lisasupport@microsoft.com
```
Comment (lizzha) [inline AutomationManager.ps1]: I think this can also be removed, and we can add the contact email(lisasupport@microsoft.com) in README of this repo

Comment (lizzha) [reply AutomationManager.ps1]: applies to other files that contain lisasupport auther

File: JenkinsPipelines/Scripts/DeletePartnerFiles.ps1 [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
```
Comment (lizzha) [inline JenkinsPipelines/Scripts/DeletePartnerFiles.ps1]: Suggest to use consistent style of headers, some files contains "#################" lines, some not
And add an empty line between header and code

File: Utilities/AddAzureRmAccountFromSecretsFile.ps1 [hunk 3]
```diff
@@ -2,8 +2,8 @@
 .SYNOPSIS
```
Comment (lizzha) [inline Utilities/AddAzureRmAccountFromSecretsFile.ps1]: here's a new style of headers

File: AutomationManager.ps1 (hunk 1 above)
Comment (juhlee-microsoft) [reply AutomationManager.ps1]: Done (7033ff1a46339c5500e7fcb6e5d4dcffb9cf9b4b)

**Comment by ader1990** [comment]
@juhlee-microsoft  AFAIK, there is no license Apache per se, maybe you are referring to Apache License, Version 2.0?

File: JenkinsPipelines/Scripts/DeletePartnerFiles.ps1 (hunk 2 above)
Comment (juhlee-microsoft) [reply JenkinsPipelines/Scripts/DeletePartnerFiles.ps1]: Done except TestScripts folder (4994d3c0864b8e32b14603604e6f43ddf8d2c946)

File: Utilities/AddAzureRmAccountFromSecretsFile.ps1 (hunk 3 above)
Comment (juhlee-microsoft) [reply Utilities/AddAzureRmAccountFromSecretsFile.ps1]: Found this comment after I made previous changes. So I'd like to make sure below final format is ok of you?
@lizzha please confirm.

##############################################################################################
# <File Name>
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the Apache License.
# Operations :
#
<#
.SYNOPSIS
	<Description>

.PARAMETER
	<Parameters>

.INPUTS
	

.NOTES
    Version: 1.0
    Creation Date:  
    Purpose/Change: 

.EXAMPLE


#>
###############################################################################################

Comment (juhlee-microsoft) [reply Utilities/AddAzureRmAccountFromSecretsFile.ps1]: Please see 893dfd927ec578f11e7c0984a6dbf30f53e36a35

File: JenkinsPipelines/Update-Jenkins-Configuration-Files.groovy [hunk 4]
```diff
@@ -1,8 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache License.
 //This script requires following parameters to be added in Jenkins job.
-//Example: 
-//GitBranch = https://github.com/iamshital/LISAv2.git
-//GitRepo = master
-//MenuFilesPath = "Shared directory between jenkins master and slave"
```
Comment (lizzha) [inline JenkinsPipelines/Update-Jenkins-Configuration-Files.groovy]: Suggest to keep the example, change the GitBranch to "https://github.com/LIS/LISAv2.git"

Comment (juhlee-microsoft) [reply JenkinsPipelines/Update-Jenkins-Configuration-Files.groovy]: Fixed in 75edda2860931a5736b1aff3d8bde679acb3ad72

File: AutomationManager.ps1 (hunk 1 above)
Comment (simonxiaoss) [reply AutomationManager.ps1]: Yes, please remove "Author : lisasupport@microsoft.com" from the individual files.

**Pull request merged**